src/DocSidebar/styles.module.css

```diff
@@ -1,12 +1,24 @@
-@value large-tablet-breakpoint-query from '~CSS/variables.module.css';
+@value large-tablet-breakpoint-query from '~libra-docusaurus/src/variables.module.css';
+
+/*
+ * For some reason, importing the variable above causes
+ * the next style to be ignored. This seems like a webpack
+ * bug, so for the sake of time I'm putting in this 
+ * quick fix (having a blank style before the real one)
+ * and hoping that the bug resolves on its own with a future
+ * update of CSS variables/webpack
+ */
```

Comment:
This is an annoying bug, but not urgent. It might exist from the fact that this is an uncompiled module. I'm going to look into it after the code freeze, as I don't think it's the most important thing now

Reply:
Sounds good, open a Jira ticket for it please.

Reply:
https://dfuzrdev.atlassian.net/browse/LIB-725